website/source/api/operator.html.md

```diff
@@ -123,3 +125,232 @@ $ curl \
     --request DELETE \
     https://nomad.rocks/v1/operator/raft/peer?address=1.2.3.4
 ```
+
+## Read Autopilot Configuration
+
+This endpoint retrieves its latest Autopilot configuration.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `GET`  | `/operator/autopilot/configuration` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | ACL Required    |
+| ---------------- | ----------------- | --------------- |
+| `NO`             | `none`            | `operator:read` |
+
+### Parameters
```

Comment:
I think we remove both of these parameters (dc should be region) but they are both covered generically on this page: https://www.nomadproject.io/api/#consistency-modes